ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -521,6 +541,7 @@ type opennebula_untouchables = {
     "datastores" ? string[]
     "vnets" ? string[]
     "users" ? string[]
+    "groups" ? string[]
```

Comment:
why are these lists? you could use a dict with the name as key. same remark for users/vnets/hosts/....